hsctl

```diff
@@ -214,12 +213,6 @@ preflight_hs() {
     cp -rf ${HS_PATH}/scripts/templates/init-defaultworker.template ${HS_PATH}/init-defaultworker
     sed -i 's!HS_SERVICE_UID!'${HS_SERVICE_UID}'!g' ${HS_PATH}/init-defaultworker
     sed -i 's!HS_SERVICE_GID!'${HS_SERVICE_GID}'!g' ${HS_PATH}/init-defaultworker
-    # Check security settings
-    if [ "${USE_SECURITY,,}" = true ]; then
-        sed -i 's/\<USE_SECURITY = False\>/USE_SECURITY = True/g' ${HS_PATH}/hydroshare/settings.py
-    else
-        sed -i 's/\<USE_SECURITY = True\>/USE_SECURITY = False/g' ${HS_PATH}/hydroshare/settings.py
-    fi
```

Comment:
@mauriel133 @pkdash I believe change of hsctl is an oversight, and the changes should not be checked in.